### ISSUE TITLE
A question about Autocomplete LSTM Tensorflow

### ISSUE THREAD

**Issue by nvlong198** [graykode/nlp-tutorial#25]
In Autocomplete We already have 
```
X = tf.placeholder(tf.float32, [None, n_step, n_class]) # [batch_size, n_step, n_class]
Y = tf.placeholder(tf.float32, [None, n_class])         
```
to guess next missing character 
1. How I can customize them to guess more than 1 character ? I don't have any idea about multiplies a tensor by tensor.
2. In  `outputs, states = tf.nn.dynamic_rnn(cell, X, dtype=tf.float32) `
Why the shape of **states** alway **(2,)**, what really **2** mean ?
Thank you for sharing the information.
![aTDpS](https://user-images.githubusercontent.com/23328191/56555441-2545c880-65bf-11e9-94c6-2616dfcf109b.png)

**Comment by graykode** [comment]
Hello. Sorry for late.
1. Its simple. you just sort output logits then pick k which number you want.
2. Please see tensorflow document https://www.tensorflow.org/api_docs/python/tf/nn/dynamic_rnn
